transport/kafka/kafka.go

```diff
@@ -71,6 +73,8 @@ func (d *KafkaDriver) Init(context.Context) error {
 	kafkaConfig.Producer.MaxMessageBytes = d.kafkaMaxMsgBytes
 	kafkaConfig.Producer.Flush.Bytes = d.kafkaFlushBytes
 	kafkaConfig.Producer.Flush.Frequency = d.kafkaFlushFrequency
+	kafkaConfig.Producer.Compression = sarama.CompressionCodec(d.kafkaCompressionType)
```

Comment:
Does it handle the case where it's empty?

Reply:
And what is the behavior if user input is not recognized?
Should we raise an error?

Reply:
We can probably let samsa raise the error if its not recognized

Reply:
Both situations IMO should be handled by samsa, we should merely provide a default value of "none" if no parameter is provided